### DISCUSSION TITLE
Can I live preview markdown/org files using the respective previewers?

### DISCUSSION THREAD

**Discussion by atharva7-g** [emacs-eaf/emacs-application-framework#937] [Q&A] [answered]
Is it possible to live preview files using `org-previewer` and `markdown previewer`, instead of using `eaf-open`? Or reload the preview?

**Comment by MatthewZMD** [comment, marked as answer]
Sorry I do not understand what you want, `eaf-open` is just a default API to open EAF supported file using an EAF app.

**Reply by atharva7-g** [reply]
Yeah, but is it possible to automatically relad the file when any changes are made, like a markdown editor?

![markdown](https://user-images.githubusercontent.com/52556680/159847128-150eb86c-326f-4c29-b6e5-7bc894a7bb68.gif)

Or maybe reload the file upon saving. Since, currently it is only possible to repeatedly use `eaf-open` preview the file.

**Reply by manateelazycat** [reply]
> Yeah, but is it possible to automatically relad the file when any changes are made, like a markdown editor?
> 
> ![markdown](https://user-images.githubusercontent.com/52556680/159847128-150eb86c-326f-4c29-b6e5-7bc894a7bb68.gif)
> 
> Or maybe reload the file upon saving. Since, currently it is only possible to repeatedly use `eaf-open` preview the file.

You can use my https://github.com/manateelazycat/auto-save, then when you type something, auto-save will trigger EAF reload preview content.

**Reply by atharva7-g** [reply]
> > Yeah, but is it possible to automatically relad the file when any changes are made, like a markdown editor?
> > ![markdown](https://user-images.githubusercontent.com/52556680/159847128-150eb86c-326f-4c29-b6e5-7bc894a7bb68.gif)
> > Or maybe reload the file upon saving. Since, currently it is only possible to repeatedly use `eaf-open` preview the file.
> 
> You can use my https://github.com/manateelazycat/auto-save, then when you type something, auto-save will trigger EAF reload preview content.

Thanks! Also is it possible to change the font of the markdown preview that opens up?

**Reply by manateelazycat** [reply]
Try https://github.com/emacs-eaf/eaf-browser/blob/0b4e57dfbf7230ea81c0e14066ef0a11950c32e4/eaf-browser.el#L164
